portal-web/test/functional/com/liferay/portalweb/paths/portlet/sitepages/SitePagesEditPage.path

```diff
@@ -72,6 +72,11 @@
 	<td>//div[contains(@class,'preview-seo-image')]//img[contains(@src,'${key_imageName}')]</td>
 	<td></td>
 </tr>
+<tr>
```

Comment:
Can you add a locator called `Label#ANY` under portalweb\paths\pathlib\uielements instead of doing this? And please mention it in a global chat like #t-qa-test-fixes so people are aware of it and can use the label

Reply:
Hi @austinchiang 
I've double-checked that path, our cases can work fine without it. I have added it to the path file before because it was written directly in the test file. But for now, I think you can remove it or I can remove it in a new pull.